src/popup.ts

```diff
@@ -6,7 +6,7 @@ import { getStorageItem, setStorageItem } from './storage';
 document.getElementById('login')?.addEventListener('click', openAuthTab);
 
 document.getElementById('go-to-options')?.addEventListener('click', () => {
-  chrome.runtime.openOptionsPage();
+  chrome.tabs.create({ url: 'chrome://extensions/?options=' + chrome.runtime.id });
```

Comment:
What's the reason for this change?

Reply:
I don't think the options page is large enough to take up one whole tab, so use this link to go to the extension detail page and pop up the options window.